include/seqan/align_parallel/parallel_align_interface.h

```diff
@@ -275,6 +275,38 @@ doWaveAlignment(ExecutionPolicy<WavefrontAlignment<TWaveSpec>, TVectorizationPol
  * Wrapper functions for calling globalAlignmentScore and localAlignmentScore with an ExecutionPolicy.
  * Note the parallel interfaces are documented as part of the standard documentation in seqan/align module.
  */
+template <typename TParallelPolicy, typename TVectorizationPolicy,
+       typename ...TArgs,
+       std::enable_if_t<std::is_same<TParallelPolicy, Serial>::value ||
```

Comment:
```suggestion
       SEQAN_FUNC_ENABLE_IF<std::is_same<TParallelPolicy, Serial>::value ||
```
?

Reply:
No, this macro only works as a function return type and placing auto there is not possible. So in order to use auto in the function return type I need to move the sfinae expression into the template argument.